test_app/tests/tests_views/tests_filter_views.py

```diff
@@ -50,3 +50,18 @@ def test_search_body_malformed(self):
         search = "potato"
         response = self.client.post(url, data=search, content_type="application/text")
         self.assertEqual(status.HTTP_415_UNSUPPORTED_MEDIA_TYPE, response.status_code)
+
+    def test_search_body_ordered(self):
+        teachers = []
+        for i in range(10, 20):
+            teachers.append(TeacherFactory(id=i, is_ghost=False, picture=None))
+        expected_teachers = [self.expected_teacher(teacher) for teacher in teachers]
+
+        for i in range(200, 210):
+            TeacherFactory(id=i, is_ghost=True, picture=None)  # noise
+
+        url = f"{self.url}/search?sort=id"
```

Comment:
Should the `sort` be provided into querystring?

Reply:
It does not work :(

![Screenshot 2024-02-20 at 10 40 23](https://github.com/flamingo-run/drf-toolkit/assets/90624354/1ef99535-452f-40e3-aee4-ccac9bf29754)
-----
This is because the `OrderingFilter` inside `rest_framework` is implemented expecting a `query_param`:

![Screenshot 2024-02-20 at 10 43 13](https://github.com/flamingo-run/drf-toolkit/assets/90624354/c31db34a-c0ac-4768-b684-c05f28ba937b)